From 4f214fd1c8f0326822c23dbcce9634e4e159c3c4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Miroslav=20Such=C3=BD?= <msuchy@redhat.com>
Date: Thu, 14 Apr 2016 15:20:08 +0200
Subject: [PATCH] add support for lzma

This is to fix
https://bugzilla.redhat.com/show_bug.cgi?id=1021625
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=506861
tarfile already support lzma so this is safe
---
 lib/debian/debfile.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/debian/debfile.py b/lib/debian/debfile.py
index 806b831..ae95b8c 100644
--- a/lib/debian/debfile.py
+++ b/lib/debian/debfile.py
@@ -28,7 +28,7 @@ from debian.deb822 import Deb822

 DATA_PART = 'data.tar'      # w/o extension
 CTRL_PART = 'control.tar'
-PART_EXTS = ['gz', 'bz2', 'xz']  # possible extensions
+PART_EXTS = ['gz', 'bz2', 'xz', 'lzma']  # possible extensions
 INFO_PART = 'debian-binary'
 MAINT_SCRIPTS = ['preinst', 'postinst', 'prerm', 'postrm', 'config']

--
2.5.5
